idl/config.x

```diff
@@ -0,0 +1,63 @@
+%#include "common.x"
+
+namespace mazzaroth
+{
+  // ContractChannelConfig stores contract channel configuration in state and is 
+  // accessible through host contract host functions
+  struct ContractChannelConfig
+  {
+    // Unique channel id which cannot be changed
+    ID channelID;
+    // Contract binary bytes
+    opaque contract<>;
```

Comment:
When we discussed what should be in the contract config we used the rough criteria "Would we ever use this in a contract?"

I don't think we would ever use the contract bytes in a contract. Is this just an exception?

Reply:
After discussion, have replaced this with the hash and restored the contract update transaction type.